python/txt2tags.py

```diff
@@ -4565,14 +4573,14 @@ def convert(bodylines, config, firstlinenr=1):
 def load_GUI_resources():
```

Comment:
How difficult would it be to strip the gui from this?

Reply:
Rather than stripping the gui, I have worked further regarding gui.
So far, just
```
python3.7 txt2tags.py --gui
```
works, but the actual job is not tested.

Reply:
For NVDA, we really don't care about the gui, but I think it is ok.